content/blog/2025-07-dev-update/index.md

```diff
@@ -0,0 +1,64 @@
++++
+type = "blog"
+title = 'July 2025 Monthly Development Update'
+date = 2025-07-15
++++
+
+This is the TLA⁺ Foundation monthly development update ([subscribe via RSS](/blog/index.xml)).
+Here we summarize the past month of development for the benefit of Foundation patrons and interested members of the community.
+We will also highlight a good bug or small feature for prospective new contributors to look at!
+If your TLA⁺ contribution was missed, worry not - we publish monthly, so it's easy to hop on the next train; open an issue [here](https://github.com/tlaplus/foundation/issues).
+
+If you're interested in getting involved in the TLA⁺ community:
+- Learn TLA⁺ starting [here](https://lamport.azurewebsites.net/tla/learning.html)!
+- Join the monthly virtual community meetings [here](https://groups.google.com/g/tlaplus/c/3CloQYEH0qQ/m/GAIxbKNgBAAJ)!
+- Read the mailing list [here](https://groups.google.com/g/tlaplus)!
+- Start hacking on the tools themselves [here](https://github.com/tlaplus/tlaplus)!
+
+### Organizational Updates
+
+- The TLA⁺ Foundation grant program is continuing on a rolling basis; please [submit a proposal](/grants/2024-grant-program/) if you have ideas to improve TLA⁺ so the Foundation can support you financially as you work on them.
+  You can find some desired projects listed [here](https://github.com/tlaplus/tlaplus/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22TLA%2B%20Foundation%20Funding%22).
+  If you're trying to figure out the next stage of your career and like the sound of being paid a stipend to work on free & open source software in the formal methods space, this is a great opportunity!
+- The [GenAI-Accelerated TLA⁺ Challenge](https://foundation.tlapl.us/challenge/) submission date was on July 3rd, and the winners will be announced at the start of August.
```

Comment:
Hmm, not sure how many submissions were received (I saw mention of one)? @lemmy, can you confirm the planning?

Reply:
There are four submissions.